source/new.rst

```diff
@@ -696,6 +686,80 @@ Notes on the verified group protocol
   Carol could send the key to further verified groups
   that Bob is not part of.
 
+Dealing with key loss and compromise
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a user looses their device
+they can setup a new device
+and regain access to their inbox.
+However they may loose their secret key.
+
+They can generate a new key pair.
+Autocrypt will distribute their new public key
+in the Autocrypt headers
+and opportunistic encryption will switch to it automatically.
+
+Verified groups will remain unreadable
+until the user verifies a contact from that group.
+Then the contact can update the key used in the group.
+This happens by sending a "vg-member-added" message
+to the group.
+Since the email address of that user remains the same
+the old key will be replaced by the new one.
+
+Implementers may decide
+wether the recipients of such key updates
+propagate them to other groups
+they share with the user in question.
+If they do this will speed up the recovery from device loss.
+However it also allows Bob-in-the-middle attacks
+that replace the originally verified keys.
+So the decision needs to be based on the threat model of the app
+and the strategy picked for verification reuse
+
+In the case of **key compromise**
+more care needs to be taken.
+Since network attackers can drop messages
+they can also drop the "vg-member-added" message
+that was meant to replace a compromised key.
```

Comment:
is this really "key compromise"? i'd understand key compromise to refer to someone getting access to the secret key.  I'd rather see it as "mitm" (when we define mitm to mean network-attacks, in contrast to "bob mitm" where a peer participates additionally in the attack.

Reply:
I tried to clarify in my latest commit.
The scenario starts with a key known or suspected to be compromised.
In that case you want to make sure the key is not used anymore... not just to prevent unreadable mail - but especially to prevent readable mail for the parties who compromised the key. Since they might be a mitm attacker you need to make sure your update actually reached everyone.